wiki/Gameplay/Input_device/Beatmania_controller/en.md

```diff
@@ -0,0 +1,11 @@
+# Beatmania Controller
+
+A **Beatmania controller** is a type of game controller that is compatible with the *[Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")* series of games. 
+Due to their nature, Beatmania controllers are most often used as input devices in [osu!mania](/wiki/Game_Modes/osu!mania).
```

Comment:
```suggestion
A **Beatmania controller** is a type of game controller that is compatible with the *[Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")* series of games. Due to their nature, Beatmania controllers are most often used as input devices in [osu!mania](/wiki/Game_Modes/osu!mania).
```
Refer to ASC for more details, but paragraphs must be seperated by one empty line. Remarkjs will also have a fit in this case.

Reply:
ah sorry, thought that was one whole paragraph